bump-periodics.sh

```diff
@@ -65,7 +65,7 @@ yq --yaml-output --in-place '.
 	| ( .releases[][] | select(.stream == "ci") | .stream ) |= "nightly"
 
 	# Set a conveniently long interval to all tests in the old periodics
-	| .tests[].interval |= "72h"
+	| .tests[].minimum_interval |= "72h"
```

Comment:
This change assumes that the OLD_VERSION job has `minumum_interval`. We should probably first remove any {interval,cron} property before adding our `minimum_interval`, to make the script more robust.

Something like:
```plaintext
| del(.tests[].interval)
| del(.tests[].cron)
| .tests[].minimum_interval |= "72h"
```

Reply:
Good idea :+1: